batch2/insert-batch-globals.py

```diff
@@ -0,0 +1,43 @@
+import sys
+import string
+import secrets
+import asyncio
+from gear import Database
+
+
+async def main():
+    scope = sys.argv[1]
+
+    worker_type = 'standard'
+    if scope == 'deploy':
+        worker_cores = 8
+        worker_disk_size_gb = 100
```

Comment:
Where do you run this script? Just in case I ever need to do it...

Reply:
I run it in the create-batch2-tables pod.